poker-app/components/AsyncStorageController.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { View, Text, AsyncStorage } from 'react-native';
```

Comment:
check if async storage is still supported. 
try import AsyncStorage from '@react-native-community/async-storage'
there was some other stuff. check instructions.

you can do it later though. not that important if you reference asyncstorage in this file only.